src/hotspot/cpu/ppc/ppc.ad

```diff
@@ -11418,90 +11418,19 @@ instruct testL_reg_imm(flagsRegCR0 cr0, iRegLsrc src1, uimmL16 src2, immL_0 zero
   ins_pipe(pipe_class_compare);
 %}
 
-instruct cmovI_conIvalueMinus1_conIvalue1(iRegIdst dst, flagsRegSrc crx) %{
-  // no match-rule, false predicate
-  effect(DEF dst, USE crx);
-  predicate(false);
+// Manifest a CmpL3 result in an integer register.
+instruct cmpL3_reg_reg(iRegIdst dst, iRegLsrc src1, iRegLsrc src2) %{
+  match(Set dst (CmpL3 src1 src2));
+  ins_cost(DEFAULT_COST * (VM_Version::has_brw() ? 4 : 5));
```

Comment:
"size" needs to be precise, but a rough estimate is sufficient for "ins_const". In this case CmpL3 has only one match rule, so matcher doesn't have a choice and cost is pointless. So I suggest to keep it more simple and make cost independent on has_brw.